server/lib/poller.js

```diff
@@ -114,6 +141,34 @@ class Poller
             log.error(`when polling accounts at startup: ${err.message}`);
         }
     }
+
+    async manageCredentialErrors(access, err) {
+        try {
+            if (err.errCode) {
+
+                 // We save the error status, so that the operations
+                 // are not fetched on next poll instance.
+                access.fetchStatus = err.errCode;
+                await access.save();
+                let request = promisify(::Bank.request);
+                let bank = await request('byUuid', { key: access.bank });
+                // Retrieve the human readable error code.
+                let error = $t(`server.email.fetch_error.${err.errCode}`);
+                let subject = $t('server.email.fetch_error.subject');
+                let content = `${$t('server.email.hello')},\n\n`;
+                content += `${$t('server.email.fetch_error.text',
+                    { bank: bank[0].name, error })}\n`;
+                content += `${$t('server.email.fetch_error.pause_poll')}\n\n`;
+                content += `${$t('server.email.signature')}`;
```

Comment:
Can you use a single template string here, on several lines, please? It is going to be easier to read.